crates/re_sdk/src/log_sink.rs

```diff
@@ -168,6 +185,8 @@ impl MemorySinkStorage {
 
 /// Stream log messages to a Rerun TCP server.
 pub struct TcpSink {
+    // used for Debug only
+    addr: std::net::SocketAddr,
```

Comment:
I'd rather we made `re_sdk_comms::Client` store its own address and expose a getter for it rather than duplicate this information here.

Reply:
Done, but instead of a getter, I used it in a `Debug` implementation on `re_sdk_comms::Client`, since adding public API isn't the focus of this PR.